op-e2e/opgeth/op_geth.go

```diff
@@ -156,21 +154,6 @@ func (d *OpGeth) AddL2Block(ctx context.Context, txs ...*types.Transaction) (*et
 		return nil, errors.New("required transactions were not included")
 	}
 
-	// if we are at Isthmus, set the withdrawalsRoot in the execution payload to the storage root of the message passer contract
-	if d.L2ChainConfig.IsIsthmus(uint64(payload.Timestamp)) {
-		var getProofResponse *eth.AccountResult
-		rpcClient := d.l2Engine.RPC
-		err := rpcClient.CallContext(ctx, &getProofResponse, "eth_getProof", predeploys.L2ToL1MessagePasserAddr, []common.Hash{}, payload.BlockHash.String())
-		if err != nil {
-			return nil, err
-		}
-		if getProofResponse == nil {
-			return nil, ethereum.NotFound
-		}
-		storageHash := getProofResponse.StorageHash
-		payload.WithdrawalsRoot = &storageHash
-	}
-
```

Comment:
Why was this removed?

Reply:
Seems like it's not needed: https://github.com/ethereum-optimism/optimism/pull/13934#discussion_r1975608590

Reply:
My understanding is: If we were validating blocks, it would be needed, but this value is determined by the block that's built, not the other way around.